backend/src/books/books.repository.ts

```diff
@@ -119,7 +115,8 @@ class BooksRepository extends Repository<Book> {
       .leftJoin(Category, 'category', 'category.id = book_info.categoryId')
       .limit(limit)
       .groupBy('book_info.id')
-      .orderBy(ordering)
+      .orderBy('title', 'ASC')
+      .orderBy(order, 'DESC')
```

Comment:
```suggestion
      .orderBy(order, 'DESC')
      .addOrderBy('title', 'ASC')
```

> If you use `.orderBy` more than once you'll override all previous `ORDER BY` expressions.

https://typeorm.io/select-query-builder#adding-order-by-expression

1. orderBy를 여러 번 실행하면 기존 orderBy가 덮어씌워지기 때문에 addOrderBy를 사용해야 할 것 같습니다.
2. 인기/생성순으로 먼저 정리하고, 이름순으로 정리하는 것이 좋을 것 같습니다.